SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -185,6 +203,22 @@ public final class SwiftMessageBar {
     tapHandler?()
   }
   
+  private func forceDismissMessage(_ message: Message) {
+    self.isMessageVisible = false
```

Comment:
again, please no `self.` in the method. And consistent 2 spaces indentation please.